FrontEnd/src/components/HeaderFooter/header/navbar/SearchBox.jsx

```diff
@@ -1,15 +1,39 @@
-import css from './SearchBox.module.css';
+import { useState } from 'react';
 import icon_search from './search-icon.svg';
+import { useNavigate } from 'react-router-dom';
+import css from './SearchBox.module.css';
+
+function SearchBox() {
+  const navigate = useNavigate();
+
+  const [searchTerm, setSearchTerm] = useState('');
+  const searchPage = 'search';
 
-function SearchBox () {
-    return (
-        <div className={css['header-search-box']}>
-            <div className={css['header-search-form']}>
-                <input className={css['header-search-form__input']} placeholder="Пошук"></input>
-            </div>
-            <span className={css['header-search-form__addon']}><img src={icon_search} alt=""/></span>
-        </div>
-    );
+  const handleSearch = (searchTerm, searchPage) => {
+    if (searchTerm.trim() !== '') {
+      navigate(`/${searchPage}/?name=${searchTerm}`);
+    }
+  };
+  return (
+    <div className={css['header-search-box']}>
+      <div className={css['header-search-form']}>
+        <input
+          className={css['header-search-form__input']}
+          value={searchTerm}
+          type="text"
+          placeholder="Пошук"
+          onChange={(e) => setSearchTerm(e.target.value)}
+        />
+      </div>
+      <button
```

Comment:
I also would expect to run a search from the keyboard on the enter press

Reply:
Can you explain it, please?

Reply:
Thank you for feedback. Done.

Reply:
I mean when I heat `Enter` button from the keyboard I would expect run a search

Reply:
Done